package.json

```
@@ -25,7 +25,7 @@
     "prepare": "[ -z \"$CI\" ] && husky install || exit 0"
   },
   "devDependencies": {
-    "lerna": "3.3.2"
+    "lerna": "4.0.0"
```

one thing - can we specify the version a bit more loosely so that we can take advantage of [`dependabot`](https://dependabot.com/) upgrades? Like so:

```suggestion
    "lerna": "^4.0.0"
```

---

I think in general it is safer to make it specific in case of breaking changes introduced in the upgraded version. But for a lib like lerna, it is probably okay.